src/components/AppLayout.js

```diff
@@ -117,28 +118,29 @@ function handleInviteClick(fullName, inviteUrl, showModal) {
       inviteUrl={inviteUrl}
     />);
 }
+
 function Header(props) {
   const { memberDetails, showModal } = props;
-  const { fullName, inviteUrl } = memberDetails || {};
+  const { fullName, inviteUrl, profileUrl } = memberDetails || {};
 
   return (
     <HeaderElem>
       <Logo />
       <span className="userSection">
         {
-          memberDetails ? (
-            <React.Fragment>
-              <button
-                className="inviteButton"
-                onClick={handleInviteClick(fullName, inviteUrl, showModal)}
-              >
-                <FontAwesomeIcon className="icon" icon={faUserPlus} />
-                Invite
-              </button>
-              <Link className="loggedInUser" to="/me">{fullName}</Link>
-            </React.Fragment>
-          )
-          : <Link className="logIn" to="/login">Log in</Link>
+          [
```

Comment:
Instead of doing this as a bunch of stuff in an array, you can just add them into the HTML separately.

```
{ inviteUrl && (
  <button ... />
)}
{ profileUrl ? (
  <Link className="profile" ... />
) : (
  <Link className="logIn" ... />
)}
```

And then you don't need the `key` attributes anymore.